### PR TITLE
Add header styling, fonts, basic sizing

### LINKED ISSUE #6: Setup header and favicon
# Header 🎨 

& Favicon 📸

### PR DESCRIPTION
**Pull request by tonyklapatch**
Adds styling for the header, adds logos, adds the fonts, and basic sizing styling.

Fix #6

Test URLs:
- Before: https://main--cigars-for-beginners--famous-smoke.hlx.page/cigars-for-beginners
- After: https://6-header-and-favicon--cigars-for-beginners--famous-smoke.hlx.page/cigars-for-beginners

### CODE REVIEW

**Comment by tonyklapatch** [comment]
Added a z-index to the mobile nav menu.

File: cigars-for-beginners/blocks/hero/hero.css [hunk 1]
```diff
@@ -16,7 +16,6 @@ main .hero-container {
   max-width: 1200px;
   margin-left: auto;
   margin-right: auto;
-  color: white;
```
Comment (codecodeio) [inline cigars-for-beginners/blocks/hero/hero.css:19]: We want the hero text to be white, correct?

**Review by codecodeio — CHANGES_REQUESTED** [review]

**Review by codecodeio — APPROVED** [review]
Header looks great!

**Review by bdeffleyfamous — APPROVED** [review]
Header looks good in desktop and mobile

**Pull request merged**